sample/front-custom-chains/index.html

```diff
@@ -0,0 +1,410 @@
+<html>
+
+<head>
+  <title>rLogin | dApp sample</title>
+  <link
+    href="https://fonts.googleapis.com/css?family=Rubik:300,300i,400,400i,500,500i,700,700i,900,900i&amp;display=swap"
+    rel="stylesheet" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+</head>
+
+<body>
+  <div style="width: 100%; height: 100%;">
+    <h2>Login</h2>
+    <button id="login">login with rLogin</button>
+    <button id="loginMetamask">login with rLogin (defaults: Metamask)</button>
+    <hr />
+    <h2>Wallet provider info</h2>
+    <p id="error" style="color: red;"></p>
+    <h3>Supported chains</h3>
+    <p id="supportedChains"></p>
+    <h3>Connected</h3>
+    <p id="connected">No</p>
+    <h3>Accounts</h3>
+    <ul id="accounts"></ul>
+    <h3>Chain id</h3>
+    <p id="chain-id"></p>
+    <hr class="be-only" />
+    <h2 class="be-only">Backend connection info</h2>
+    <h3 class="be-only">Access token</h3>
+    <p id="access-token" class="be-only"></p>
+    <h3 class="be-only">Refresh token</h3>
+    <p id="refresh-token" class="be-only"></p>
+    <hr />
+    <h2>Actions</h2>
+    <button id="showInfo" disabled>show wallet info</button>
+    <br /><br />
+    <button id="changeNetwork" disabled>change network</button>
+    <br /><br />
+    <button id="send-tx" disabled>send</button>
+    <p id="tx"></p>
+    <button id="sign" disabled>sign</button>
+    <p id="signature"></p>
+    <p id="recovery"></p>
+    <button id="sign-typed-data-v1" disabled>sign typed data (v1)</button>
+    <button id="sign-typed-data" disabled>sign typed data (v4)</button>
+    <p id="typed-signature"></p>
+    <p id="typed-recovery"></p>
+    <hr />
+    <button id="disconnect" disabled>Disconnect</button>
+    <button id="reset">Reset</button>
+  </div>
+
+  <script src="http://localhost:3005/index.js"></script>
```

Comment:
I know this is how the other samples are done, but Sonar has blocked you on this one, so we need to at least fix it here.